src/Resources/views/webprofiler.html.twig

```diff
@@ -62,11 +62,9 @@
             {{ include('@Httplug/Icon/httplug.svg') }}
         </span>
         <strong>Httplug</strong>
-        {% if collector.failedStacks|length %}
-            <span class="count">
-                <span>{{ collector.failedStacks|length }}</span>
-            </span>
-        {% endif %}
+        <span class="count">
+            <span>{{ collector.failedStacks|length ?:  collector.stacks|length}}</span>
```

Comment:
isn't this misleading when we show the stacks count or the failed count?

i guess `stacks` implicitly is the successful stacks? if that is correct, i'd show the sum. if it is all, i'd always show the stacks length. or we could do `{{ stacks|length }}{% if failedStacks|length %} (failedStacks|length){%endif%}`

Reply:
No, stacks is all of them, including unsuccessful. Reason I change failedStacks/stacks is because when there is failedStacks, background changes to red - at that point it would be misleading to highlight number representing all of the requests as red, that's why at that point only number of failed requests is made red.

Reply:
ah. how about still outputting the total count along with the error count? `2 (6)` or something like that?

Reply:
i checked what the doctrine profiler does, but it seems to not mark queries as failed in the pill on the toolbar, so it is different.

Reply:
Yeah because it's not acceptible that database query fails, that would be an exception. Most similar to this would be Logs tab.

How about {successful}/{total} number? Eg. 3/3 in case all are successful, or 2/3 in case one failed.

Reply:
yeah, i like that one, should be pretty self-explanatory to read